examples/nodejs-simple/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "dependencies": {
-    "@google-cloud/agones-sdk": "1.5.0"
+    "@google-cloud/agones-sdk": "1.6.0"
```

Comment:
Is it worth also incrementing the example version?

https://github.com/googleforgames/agones/blob/master/examples/nodejs-simple/Makefile#L31

Outside of that, this is good to go.